api/halls/halls.go

```diff
@@ -0,0 +1,103 @@
+package halls
+
+import (
+	"database/sql"
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+
+	halldb "github.com/darkjedidj/cinema-service/internal/repository/halls"
+	"github.com/gorilla/mux"
+)
+
```

Comment:
```
type Handler struct {
    Repo: halldb.Repository
}
```

Reply:
what does ":" mean? didnt see such syntax, and why not using pointer on Repository struct there?

Reply:
Use pointer, ignore ":", i was writing it while doing smth else